### PR TITLE
[doc](doris-future)Add doc for doris future

### PR DESCRIPTION
**Pull request by Yukang-Lian**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Yukang-Lian** [comment]
run buildall

File: docs/en/community/doris-future.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+under the License.
+-->
+
+# doris future project
+
+## introduction
+
+The doris future project is a growth plan for novice developers, aiming to allow more developers to quickly get started with doris. The community warmly welcomes all interested developers to join.
+
+## 参与方式
```
Comment (gitccl) [inline docs/en/community/doris-future.md:33]: should use english

**Comment by Yukang-Lian** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**